.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
https://github.com/Theophileaseh/Leaderboard/blob/903732b23d8c0b6bf87f5f6d0f5869aaffec1fb5/package.json#L6-L10

- [ ] I can not start your app from my computer even after running `npm install` then `npm start` but I'm getting this error message instead ⬇️  Kindly fix this

![npm](https://user-images.githubusercontent.com/79058364/161992972-490dd59a-4355-4643-822f-bfbccdd74642.png)